libertree-frontend-ramaze/view/docs/en_GB/custom-css-file.markdown

```diff
@@ -0,0 +1,54 @@
+# Custom-CSS-File
+You can copy paste the code below in to your settings. Please put behind the semicolons a number in hexadecimal for the color. 
```

Comment:
No it's not, but if I add a default color it wouldn't look good though. If you want to have default colors which colors would you prefer? Is black and white okay?